fugue_test/execution_suite.py

```diff
@@ -553,6 +553,7 @@ def test_limit(self):
             d = e.limit(a, n=1, presort="a asc, b desc", partition_spec=ps)
             f = e.limit(a, n=1, presort=None, partition_spec=ps2)
             g = e.limit(a, n=2, presort="a desc", na_position="last")
+            h = e.limit(a, n=2, presort="a", na_position="first")
```

Comment:
This test is pretty similar to c. Are you test if presort works with just a letter?

Reply:
no, this is to cover all code you wrote, you didn't cover this case, so the test coverage dropped to 99%